index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="app.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+
+</head>
+
+<body>
+    
+    <header class="navlogo">
+        <img src="images/My Logo.png" alt="Mylogo">
+
+        <u class="nav-icon">
+            <li class="Hamburger">
+                <img src="images/Union.png" alt="menuicon">
+                <a href="#">
+                    <i class="fa-thin fa-bars"></i>
+                </a>
+            </li>
+        </u>
```

Comment:
- [ ] As explained by the previous reviewer, the `nav` tag must be inside the `header` tag but I can see you have removed it. Please know that always the hamburger icon must be wrapped into the `nav` tag. Kindly make sure to change the `u` tag to the `nav` tag in order to follow the HTML best practices and meet the project requirements.